### PR TITLE
fix(travis): run deploy script on tagged commit

### PR DESCRIPTION
**Pull request by Lehren**
Run deploy script after build success.

### CODE REVIEW

**Comment by remcohaszing** [comment]
Wouldn't it be better to use a [script deployment](https://docs.travis-ci.com/user/deployment/script/)?

This allows for better control on when to deploy. I assume we only want to deploy on tagged commits.

**Comment by Lehren** [comment]
Done

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +13,7 @@ before_install:
   - sh -e /etc/init.d/xvfb start
 deploy:
   provider: npm
+  script: ./deploy.sh
```
Comment (remcohaszing) [inline .travis.yml]: This doesn't set it as a deploy target. See https://docs.travis-ci.com/user/deployment#Deploying-to-Multiple-Providers

Comment (Lehren) [inline .travis.yml]: Oh I see. Fixed it.

File: .travis.yml [hunk 2]
```diff
@@ -19,3 +18,8 @@ deploy:
     secure: KlsdLI5KOLYVhkuQTAxxHlBMNcWtKJ3mCoX66jy1vv4CaIaYBrtgpU8BBPoR8zpQOfvpr7LwaOmeSSmCvfsOC3Ai+0XtmJNBeotnpCkHOz5Z7+vFpUmpDjWiCtIkerlZZdCFBIv1t8Lz870RVoTyevZo7txHcr6BD4QMEJcS8siZmGNLOSiUeoJXkpd9e4VNHKjodbm8stCWQjdIQ4f85B/EvGPzS3vosh8M6W++w1j1vNeJXCGfjoI+bfDr1sg0P31h2KBSWmTF+idFMSEWPgbPwq0/rrcUwTrrpaA2mbarNG5aDLRIdmUFdw+xw3TW/TE6euy6SVNFOcE+Wy8BKgOsZLVtz9s6TWB0eHcfoWg5nEwbRwu1FHHItmVMP9nW7aX7mKBlXBnXh5egekw4m6vGwUEe9ayhJs0xXz8vyrvkwJvAtDmx0eGaCLupBbHJRnjZvlLqU+2HeUMxNz3t/MbXdbytazyA5TDiBp5ZSJ9srCoCby0mnFjkQxmgFB6XbEpVz6Zo1e0hOwdP5P7iWxbKzBGvwgK2QjZDhLpCctvTWPGNmO0bUZMbCdVPPRcfIKe2m83CTvVbUy47vzreeiaVhx+Wef0MUr/cm1SvDOoNaIWb2j5eVHUvbOr5+L1uV0peR6oO26xOVmQZzyMBjC+K9vfiGeHUO3t4072C4nk=
   on:
     tags: true
+
+  provider: script
+  script: ./deploy.sh
+  on:
+    tags: true
```
Comment (remcohaszing) [inline .travis.yml]: Add a newline to the end of text files.

File: .travis.yml [hunk 3]
```diff
@@ -19,3 +18,8 @@ deploy:
     secure: KlsdLI5KOLYVhkuQTAxxHlBMNcWtKJ3mCoX66jy1vv4CaIaYBrtgpU8BBPoR8zpQOfvpr7LwaOmeSSmCvfsOC3Ai+0XtmJNBeotnpCkHOz5Z7+vFpUmpDjWiCtIkerlZZdCFBIv1t8Lz870RVoTyevZo7txHcr6BD4QMEJcS8siZmGNLOSiUeoJXkpd9e4VNHKjodbm8stCWQjdIQ4f85B/EvGPzS3vosh8M6W++w1j1vNeJXCGfjoI+bfDr1sg0P31h2KBSWmTF+idFMSEWPgbPwq0/rrcUwTrrpaA2mbarNG5aDLRIdmUFdw+xw3TW/TE6euy6SVNFOcE+Wy8BKgOsZLVtz9s6TWB0eHcfoWg5nEwbRwu1FHHItmVMP9nW7aX7mKBlXBnXh5egekw4m6vGwUEe9ayhJs0xXz8vyrvkwJvAtDmx0eGaCLupBbHJRnjZvlLqU+2HeUMxNz3t/MbXdbytazyA5TDiBp5ZSJ9srCoCby0mnFjkQxmgFB6XbEpVz6Zo1e0hOwdP5P7iWxbKzBGvwgK2QjZDhLpCctvTWPGNmO0bUZMbCdVPPRcfIKe2m83CTvVbUy47vzreeiaVhx+Wef0MUr/cm1SvDOoNaIWb2j5eVHUvbOr5+L1uV0peR6oO26xOVmQZzyMBjC+K9vfiGeHUO3t4072C4nk=
   on:
     tags: true
+
```
Comment (remcohaszing) [inline .travis.yml]: The docs state a list of configurations should be passed. Now a yaml object with duplicate keys is passed. I'm not even sure if this is valid yaml.

File: .travis.yml (hunk 2 above)
Comment (Lehren) [inline .travis.yml]: Done

File: .travis.yml (hunk 3 above)
Comment (Lehren) [inline .travis.yml]: You mean the two `on:` tags?

I think it could be valid. You can have multiple api keys for different deployers right?

**Pull request merged**